.github/workflows/frameworks-on-pr.yml

```diff
@@ -564,24 +568,25 @@ jobs:
       - name: Upload test coverage to Codecov
         uses: codecov/codecov-action@v1.0.12
 
-  spacy_integration_tests:
-    needs: changes
-    if: ${{ needs.changes.outputs.spacy == 'true' }}
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0 # fetch all tags and branches
-      - name: Setup python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
-      - name: Install test dependencies
-        run: ./ci/install_dependencies.sh.cmd
-      - name: Run tests
-        run: ./ci/integration/frameworks/run_spacy_tests.sh
-      - name: Upload test coverage to Codecov
-        uses: codecov/codecov-action@v1.0.12
+#  TODO: fix me
```

Comment:
@parano why do we disable spacy tests? I think it passed right?

Reply:
@aarnphm I did that by mistake, I will uncomment this in a follow up PR

Reply:
👍🏼